src/utils/src/FileIo.c

```diff
@@ -23,8 +23,18 @@ STATUS readFile(PCHAR filePath, BOOL binMode, PBYTE pBuffer, PUINT64 pSize)
     CHK(fp != NULL, STATUS_OPEN_FILE_FAILED);
 
     // Get the size of the file
+
+    // Use Windows-specific _fseeki64 and _ftelli64 as the traditional fseek and ftell are non-compliant on systems that do not provide
+    // the same guarantees as POSIX. On these systems, setting the file position indicator to the
+    // end of the file using fseek() is not guaranteed to work for a binary stream, and consequently,
+    // the amount of memory allocated may be incorrect, leading to a potential vulnerability.
+#if defined _WIN32 || defined _WIN64
+    _fseeki64(fp, 0, SEEK_END);
```

Comment:
Is it the case for Windows that we should *always* use `_fseeki64` instead of `fseek` regardless of arguments?  If so I think we should adjust the #define here:

https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/57637ea593f4b43c509413a44d993ed08d7f2616/src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h#L900

Witha n if/def for Windows to have 

```
#ifndef FSEEK
#if defined _WIN32 || defined _WIN64
#define FSEEK _fseeki64
#else
#define FSEEK fseek
#endif
#endif

Reply:
Same for `FTELL`

Reply:
Done!